config/confighttp/README.md

```diff
@@ -20,6 +20,7 @@ README](../configtls/README.md).
 - [`read_buffer_size`](https://golang.org/pkg/net/http/#Transport)
 - [`timeout`](https://golang.org/pkg/net/http/#Client)
 - [`write_buffer_size`](https://golang.org/pkg/net/http/#Transport)
+- `compression`: Compression type to use among `gzip`, `zlib`, `deflate`, `snappy` and `zstd`
```

Comment:
Should we be making a recommendation here to users? According to the notes from the SIG meeting from Nov 3rd, `zstd` would be the 1st option, with `snappy` being the 2nd.

Reply:
changed the order and added recommendation.